src/main/resources/migration/common/V49__add_address_table_and_licence_link_tables.sql

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
have merge this into the same PR as I was fed up of updating both branches all the time